docs/basics.txt

```diff
@@ -556,7 +585,7 @@ IMPORTANT: Do not use bin/janusgraph.sh.  That starts the default configuration,
 The default configuration described in <<server-getting-started>> is a WebSocket configuration.  If you want to alter the default configuration in order to use JanusGraph Server as a HTTP endpoint for your JanusGraph database, follow these steps:
 
 .To Configure JanusGraph Server For Rest
```

Comment:
You didn't change this line but should it be "for REST"?

Reply:
It should be 'for HTTP'. There is no REST interface.

Reply:
@robertdale is always on HTTP vs REST patrol duty :)